webapp/src/components/App/Studies/utils.ts

```diff
@@ -20,6 +20,16 @@ export interface StudyTreeNode {
   children: StudyTreeNode[];
 }
 
+export interface NonStudyFolderDTO {
```

Comment:
Rename `class NonStudyFolder(AntaresBaseModel)` with 'DTO' suffix in Python code